app/Http/Traits/UploadFileTrait.php

```
@@ -9,23 +9,37 @@ trait UploadFileTrait
-        return $fileLocation . $hashname . "/" . $originalName;
+        $outputFileName = $newName ?? $file->getClientOriginalName();
+        $file->storePubliclyAs($fileLocation . $hashname, $outputFileName, 'public');
+        return $fileLocation . $hashname . "/" . $outputFileName;
+    }
+
+    public function uploadFileStoragedisk($file, $fileLocation, $filename)
+    {
+        if (!file_exists($fileLocation)) {
+            Storage::disk('public')->makeDirectory($fileLocation, 0755, true);
+        }
+        $hashmake = Hash::make('secret');
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Hash' in method 'uploadFileStoragedisk'. (undefined)

(StaticAccess)